CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+Once you make the necessary changes on this branch, you should file a pull request to incorporate your changes into the main The Software Gardening Almanac repository.
+
+The content and description of your pull request are directly related to the speed at which we are able to review, approve, and merge your contribution into The Software Gardening Almanac.
+To ensure an efficient review process please perform the following steps:
+
+1. Follow all instructions in the [pull request template](https://github.com/software-gardening/almanac/blob/main/.github/PULL_REQUEST_TEMPLATE.md)
+1. Triple check that your pull request is adding _one_ specific feature. Small, bite-sized pull requests move so much faster than large pull requests.
+1. After submitting your pull request, ensure that your contribution passes all status checks (e.g. passes all tests)
+
+Pull request review and approval is required by at least one project maintainer to merge.
+We will do our best to review the code additions in a timely fashion.
+Ensuring that you follow all steps above will increase our speed and ability to review.
+We will check for accuracy, style, code coverage, and scope.
+
+### Git commit messages
+
+For all commit messages, please use a short phrase that describes the specific change.
+For example, “Add feature to check normalization method string” is much preferred to “change code”.
+When appropriate, reference issues (via `#` plus number) .
+
+## Development
+
+### Overview
+
+The Software Gardening Almanac is primarily written in [Python](https://www.python.org/) through [Jupyter Book](https://jupyterbook.org/) with related environments managed by Python [Poetry](https://python-poetry.org/).
+[Node](https://nodejs.org) and [NPM](https://www.npmjs.com/) dependencies are also used to assist development activities.
+We use [GitHub actions](https://docs.github.com/en/actions) for [CI/CD](https://en.wikipedia.org/wiki/CI/CD) procedures such as automated tests.
+
+### Getting started
+
+To enable local development, perform the following steps.
+
+1. [Install Python](https://www.python.org/downloads/) (suggested: use [`pyenv`](https://github.com/pyenv/pyenv) for managing Python versions)
+1. [Install Poetry](https://python-poetry.org/docs/#installation)
+1. [Install Poetry environment](https://python-poetry.org/docs/basic-usage/#installing-dependencies): `poetry install`
+1. [Install Node](https://nodejs.org/en/download) (suggested: use [`nvm`](https://github.com/nvm-sh/nvm) for managing Node versions)
+1. [Install Node environment](https://docs.npmjs.com/cli/v9/commands/npm-install): `npm install`
+
+### Development Tasks
+
+Common development tasks are defined using [Poe the Poet](https://poethepoet.natn.io/), which helps simplify repeated use of commands.
+Poe the Poet is included as a Python Poetry dev group dependency and tasks may be used through the Poetry environment.
```

Comment:
```suggestion
We use [Poe the Poet](https://poethepoet.natn.io/) to define common development tasks, which simplifies repeated  commands.
We include Poe the Poet as a Python Poetry dev group dependency, which users access through the Poetry environment.
```

It might be helpful to provide an example of "repeated commands"

Reply:
Ah! You do provide a link to the available tasks.

Reply:
And another concrete example, great!

Reply:
Thanks! 🙂